src/ui/EditorNS/editor.cpp

```diff
@@ -23,53 +24,61 @@ namespace EditorNS
     Editor::Editor(QWidget *parent) :
         QWidget(parent)
     {
-
-        QString themeName = NqqSettings::getInstance().Appearance.getColorScheme();
-        if (themeName == "") {
-            themeName = "default";
-        }
-
-        fullConstructor(themeFromName(themeName));
+        fullConstructor();
```

Comment:
I think the content of fullConstructor() can now be moved within the real constructor.